package.json

```diff
@@ -116,38 +116,38 @@
     "bundlewatch": "^0.3.3",
     "clean-css-cli": "^5.5.2",
     "cross-env": "^7.0.3",
-    "eslint": "^8.10.0",
+    "eslint": "^8.11.0",
     "eslint-config-xo": "^0.40.0",
     "eslint-plugin-import": "^2.25.4",
     "eslint-plugin-unicorn": "^41.0.0",
     "find-unused-sass-variables": "^4.0.3",
     "globby": "^11.0.4",
     "hammer-simulator": "0.0.1",
-    "hugo-bin": "^0.81.3",
+    "hugo-bin": "^0.82.2",
     "ip": "^1.1.5",
     "jquery": "^3.6.0",
     "karma": "^6.3.17",
     "karma-browserstack-launcher": "1.4.0",
-    "karma-chrome-launcher": "^3.1.0",
+    "karma-chrome-launcher": "^3.1.1",
     "karma-coverage-istanbul-reporter": "^3.0.3",
     "karma-detect-browsers": "^2.3.3",
     "karma-firefox-launcher": "^2.1.2",
     "karma-jasmine": "^4.0.1",
     "karma-jasmine-html-reporter": "^1.7.0",
-    "karma-rollup-preprocessor": "7.0.7",
-    "lockfile-lint": "^4.6.2",
+    "karma-rollup-preprocessor": "7.0.8",
```

Comment:
Didn't caught you earlier. :/

This should be reverted cause of this issue https://github.com/jlmakes/karma-rollup-preprocessor/issues/75

Reply:
I know, I thought I reverted it, but apparently I didn't. Feel free to make a PR to downgrade this again and add it to the `update-dependencies` exclusion for now.